src/multibody/liegroup/special-euclidean.hpp

```diff
@@ -783,6 +791,14 @@ namespace pinocchio
       qout_.template tail<4>().normalize();
     }
 
+    template <class Config_t>
+    static bool isNormalized_impl(const Eigen::MatrixBase<Config_t>& qin,
+                           const Scalar prec)
+    {
+      Scalar norm = Scalar(qin.template tail<4>().norm());
+      return (std::abs(norm - Scalar(1.0)) < prec );
```

Comment:
Don't use std::abs here, but rather using std::abs. It will be helpful for Scalar overloading.

Reply:
Ok

Reply:
I am sorry, you said
```
Don't use std::abs here, but rather using std::abs
```
Not clear. I assume you meant I should just use `abs`, without `std::`?

Reply:
You should do something like:
```c++
using std::abs;
return abs(value);
```